src/import/pull-common.c

```diff
@@ -355,7 +369,10 @@ static int verify_one(PullJob *checksum_job, PullJob *job) {
                 return -EBADMSG;
         }
 
-        line = strjoina(job->checksum, " *", fn, "\n");
+        if (endswith(checksum_job->url, ".sha256"))
+                line = strjoina(job->checksum, "  ", fn, "\n");
+        else
+                line = strjoina(job->checksum, " *", fn, "\n");
```

Comment:
so, do i get this right, suse style checksum files do not use the "binary" mode of the sha256 command? That's nasty.... Is there any chance to get this fixed in suse?

If not, I'd prefer if our code would always try the binary verificatoin first (i.e. with the asterisk), and if that fails, we should retry with the tetx verification (i.e. without the asterisk). That's a tiny bit more expensive, but more robust, and increases compatibility with whatever people want to set up.